pkgs/applications/office/espanso/default.nix

```diff
@@ -3,67 +3,114 @@
 , rustPlatform
 , pkg-config
 , extra-cmake-modules
+, dbus
 , libX11
 , libXi
 , libXtst
 , libnotify
+, libxkbcommon
 , openssl
 , xclip
 , xdotool
+, setxkbmap
+, wl-clipboard
+, wxGTK32
 , makeWrapper
 , stdenv
 , AppKit
 , Cocoa
 , Foundation
+, IOKit
+, waylandSupport ? false
+, testers
+, espanso
 }:
-
+let
+  x11Support = !waylandSupport && stdenv.isLinux;
+in
 rustPlatform.buildRustPackage rec {
   pname = "espanso";
-  version = "0.7.3";
+  version = "2.1.8";
 
   src = fetchFromGitHub {
-    owner = "federico-terzi";
+    owner = "espanso";
```

Comment:
Convert `pname` references to "espanso" too.

Reply:
Done 👍 What's the reason to not use the pname here?

Reply:
It's the same value, but semantically different. At least that is my reason for disliking that pattern.